packages/react/src/accordion-content/AccordionContent.tsx

```diff
@@ -13,7 +13,7 @@ export const AccordionContent = forwardRef<
   return (
     <Box asChild ref={ref} {...styles.content({}, className)} {...props}>
       <RadixAccordion.Content>
-        <Box color="fg.default" fontSize="md" pb="md">
+        <Box color="fg.default" fontSize="md">
```

Comment:
we have to apply padding to this inner box instead of the outer content because opening/closing the item applies height animation to this outer component - and the padding will throw off the animation - similar to the other flexbox/gap change - it'll introduce a "jump" to the animation.

you can clearly see the jumping in this recording when you remove text from the content:

https://github.com/user-attachments/assets/3ac2be35-a6fa-4779-b78a-c56cf1d8df1f